test/coverlet.core.tests/InstrumenterHelper.cs

```diff
@@ -363,4 +432,23 @@ public void LogWarning(string message)
             File.AppendAllText(_logFile, message + Environment.NewLine);
         }
     }
+
+    class InstrumentationHelperForDebugging : InstrumentationHelper
```

Comment:
@shargon  I did this to avoid restore and make instrumentation completely debuggable